lxd/device/disk.go

```diff
@@ -253,6 +253,10 @@ func (d *disk) validateConfig(instConf instance.ConfigReader) error {
 					return fmt.Errorf("Custom block volumes cannot have a path defined")
 				}
 			}
+
```

Comment:
@monstermunchkin could make this part of the related block above via an `else if` here rather than a separate if. Keeps all that similar validation logic together that way.

Reply:
Fixed